CI/appveyor.set-build-info.ps1

```diff
@@ -1,7 +1,12 @@
 cd $Env:APPVEYOR_BUILD_FOLDER
 
 if ($Env:APPVEYOR_REPO_TAG -eq "false") {
-  $Env:MUDLET_VERSION_BUILD = "-testing"
+  # The only scheduled Appveyor builds are public test builds
+  if ($Env:APPVEYOR_SCHEDULED_BUILD -ne "false") {
```

Comment:
Might be this variable is only defined for scheduled builds? Maybe check `$Env:APPVEYOR_SCHEDULED_BUILD -eq "True"`?

Reply:
Sure, updated.